pkg/vsphere/vm/vm.go

```diff
@@ -208,3 +208,15 @@ func (vm *VirtualMachine) WaitForKeyInExtraConfig(ctx context.Context, key strin
 	}
 	return detail, nil
 }
+
+func (vm *VirtualMachine) LoadName(ctx context.Context) (string, error) {
```

Comment:
This should be just `Name()`  - that's the Go convention and is consistent with govmomi